molecule/default/molecule.yml

```diff
@@ -12,7 +13,8 @@ platforms:
     image: "geerlingguy/docker-${MOLECULE_DISTRO:-debian11}-ansible:latest"
     command: ${MOLECULE_DOCKER_COMMAND:-""}
     volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
```

Comment:
Since this is not running on runner01, there's no security concern for this particular point. From my point of view, this is eventually an issue to have this running on a critical environment, and I would recommend to the --cgroupns option when running the container against Docker.